content/post/stats_at_work.md

```diff
@@ -0,0 +1,95 @@
+
+DS: The exact definition of the p-value is not super helpful, but it basically quantifies the probability of seeing what we saw - namely that A is better than B - if in fact A was the same as B! In other words, if A and B were equal, then how unlikely was our result?   
+MA: So a small p-value means that if A and B were equal, then our result would be highly unlikely. And this leads us to conclude that the result of the test can be trusted? It's accurate?
+
+DS: Kinda yeah. To be clear, there are different ways to quantify this kind of stuff. This type of analysis, called hypothesis testing, is the most common one used by the vast majority of scientists.   
+MA: Okay. So at the end of the test, you calculate the p-value and saw that it was under 5%. 
+
+DS: Right.  
+MA: A's overall conversion rate was 21% and B's 20.1%. So we're confident that A is bigger than B, cause we got a p-value less than 5%. But what if A's conversion rate was 20.2% instead? Or 25%? Don't we care about how much bigger A is?
+
+DS: Absolutely. There's some difference between A and B that we care about - say it's 0.5%. This is called the effect size. If the difference between A and B is less than 0.5%, then we say they're effectively equal. The p-value calculation takes this into account.   
+MA: Oh cool - so there's a link between p-value and effect size.
+
+DS: Yup. And the sample size.  
+MA: Right that was the next thing I was curious about. How did you know what the right sample size was?
+
+DS: Yeah, so that's all about figuring out how many users we needed to be confident in our conclusion.  
+MA: Is that the confidence interval?
+
+DS: Sort of. It's related. The confidence interval is more about telling us the interval around our estimate. Remember, we're estimating the difference between A's conversion rate and B's conversion rate.  
+MA: How does sample size fit in? Is there a formula for that?
+
+DS: It's all one formula actually! Given a significance threshold, an effect size and a power level, I can calcu-  
+MA: Wait what's power?
+
+DS: Oh sorry. Power is a term that's often overlooked because it's usually baked into the calculations. Power is sometimes used in place of sample size because they are very much linked, even though they are different concepts. Let's take a step back. Recall there are two things we want to check for in this A/B test. First, that A is truly bigger than B by at least the amount we care about (the effect size). Second, we want to be confident in our conclusion.  
+MA: And that we have a big enough sample size. 
+
+DS: Yeah - confidence in our estimate and sample size are effectively the same thing. Power is just another way to measure the confidence in our p-value calculation. When I have a small sample, the p-value that I calculate might not be very reliable. You can imagine that the error bars around my p-value estimate are very wide.  
```

Comment:
This doesn't line up with my understanding of power (which may be wrong!). I've always understood that the p-value (or `alpha`) was the probability of making a type-1 error while power was the probability of making a type-2 error. 

This idea of "measuring the confidence in our p-value calculation" doesn't sound right to me, but am happy to be corrected!

Reply:
Right. So the p-value is the probability of making a type-1 error and power is 1 - P(type II error).

I'm playing fast and loose with terminology for the sake of the marketing analyst and trying to connect the formula with the intuition. When I say "Power is just another way to measure the confidence in our p-value calculation" what I mean is that increasing sample size (and hence power, holding all else constant) is a way to ensure that, when a true difference exists, our p-value is more likely to be rejected (i.e., it gets smaller). So I think I'm missing the bit about the Null being false.  Does that make sense?